src/App.js

```diff
@@ -1,84 +1,57 @@
-import React from "react";
-import "./App.css";
+import React, { useState, useEffect } from 'react';
+import './App.css';
+import CurrentWeather from './components/currentWeather';
+import WeatherForecast from './components/WeatherForecast';
+import { filterForecast } from './utils/functions';
+import SelectCity from './components/selectCity';
+
+const myId = process.env.REACT_APP_MY_API_ID;
 
 const App = () => {
+  const [weather, setWeather] = useState(null);
+  const [city, setCity] = useState('Prague');
+  const [forecast, setForecast] = useState(null);
+
+  const fetchWeather = (city) => {
+    fetch(
+      `https://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&APPID=${myId}`,
+    )
+      .then((response) => response.json())
+      .then((data) => {
+        setWeather(data);
+      });
+  };
+
+  const fetchForecast = (city) => {
+    fetch(
+      `https://api.openweathermap.org/data/2.5/forecast?q=${city}&units=metric&APPID=${myId}`,
+    )
+      .then((response) => response.json())
+      .then((data) => {
+        setForecast(filterForecast(data.list));
+      });
+  };
+
+  const handleChangeCity = (c) => {
+    setCity(c);
+  };
+
+  useEffect(() => {
+    fetchWeather(city);
+    fetchForecast(city);
+  }, [city]);
+
+  console.log(weather);
+  console.log(forecast);
```

Comment:
console.logy většinou v aplikaci nenecháváme

Reply:
na tohle si musim davat pozor, to tam vzdy zapomenu

Reply:
Tak on to není big deal, tady to ničemu nevadí a měla bys na to normálně nástroje co by tě upozornily.